crypto/fipsmodule/ec/p384.c

```diff
@@ -162,7 +162,7 @@ static void p384_felem_cmovznz(p384_limb_t out[P384_NLIMBS],
 
 // NOTE: the input and output are in little-endian representation.
 static void p384_from_generic(p384_felem out, const EC_FELEM *in) {
-  p384_felem_from_bytes(out, in->bytes);
+  p384_felem_from_bytes(out, (const uint8_t *)in->words);
```

Comment:
can you please add a TODO and maybe open a SIM for adding `p384_felem_from/to_words` as Boring did in P-256  so we can remove all the casts in P-384/521 files?